### DISCUSSION TITLE
Site is down after changing DNS records

### DISCUSSION THREAD

**Discussion by amfonte** [vercel/community#7253] [Help]
### Page to Investigate

anthonyfonte.com

### Steps to Reproduce

I've been tracking the DNS records propagating over to Vercel and everything seems to be fine, but my website (anthonyfonte.com) is still down.

**Comment by amyegan** [comment]
Hey @amfonte! The site appears to be working now. Are you still having trouble with it?

**Reply by amfonte** [reply]
Still down on my end!
<img width="1678" alt="SCR-20240621-mxtq" src="https://github.com/vercel/community/assets/158623235/7cfe2830-2e16-4601-bf61-4c127a1ff1b6">

**Reply by amfonte** [reply]
What I don't understand is that it works on my phone and tablet, but not my laptop. I've tried clearing my cache and cookies.

**Reply by amyegan** [reply]
Does it work in incognito mode or with extensions disabled?

**Reply by amfonte** [reply]
No, neither of those have an impact.

**Reply by amfonte** [reply]
I tested an older laptop I own and the site is up. If I'm accessing the site on a work laptop, is there something I can look into as to why it all of a sudden isn't loading? I have been able to visit the site fine since I launched it in February, but ever since I had to update the DNS records, it refuses to load on this work laptop.

**Reply by amfonte** [reply]
Is there a way to test if my laptop is blocking my newly updated DNS records?

**Reply by amfonte** [reply]
I noticed that my domain registrar had IP addresses for the DNS records to point to. But now that I updated those with the company that handles my domain, there are no IP addresses for the records to point to. Is that something Vercel handles? Do I need to set this up in my Vercel account?
<img width="731" alt="SCR-20240621-ppxz" src="https://github.com/vercel/community/assets/158623235/2d05756c-414c-402f-bf4f-d7021da16f27">
<img width="718" alt="SCR-20240621-ppyz" src="https://github.com/vercel/community/assets/158623235/af7a5ffb-2b44-4dc7-9bcd-30e75df0de6d">

[a comment by an automated account is not shown]
